configuration/publishing.gradle

```diff
@@ -17,6 +17,9 @@ apply plugin: 'maven'
 apply plugin: 'signing'
 apply plugin: 'maven-publish'
 
+version = project.ext.VERSION_NAME
+group = POM_GROUP
```

Comment:
In the previous change, I validated some of the contents of the aar file, but apparently not all the fields in the pom. Can you double check the version in the pom and in the AndroidManifest.xml?

```
mkdir explode
cd explode
unzip core-kotlin.aar
cat AndroidManifest.xml
```

Reply:
Here are those files in my ~/.m2 mavenLocal directory, after making the change in this PR.  I think it's all correct!

AndroidManifest.xml:
```
<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    package="com.amplifyframework.kotlin"
    android:versionCode="100"
    android:versionName="0.1.0" >

    <uses-sdk
        android:minSdkVersion="16"
        android:targetSdkVersion="30" />

</manifest>
```


core-kotlin-0.1.0.pom:

```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <modelVersion>4.0.0</modelVersion>
  <groupId>com.amplifyframework</groupId>
  <artifactId>core-kotlin</artifactId>
  <version>0.1.0</version>
  <packaging>aar</packaging>
  <dependencies>
    <dependency>
      <groupId>org.jetbrains.kotlin</groupId>
      <artifactId>kotlin-stdlib</artifactId>
      <version>1.4.30</version>
    </dependency>
    <dependency>
      <groupId>org.jetbrains.kotlinx</groupId>
      <artifactId>kotlinx-coroutines-core</artifactId>
      <version>1.4.2</version>
    </dependency>
    <dependency>
      <groupId>androidx.core</groupId>
      <artifactId>core-ktx</artifactId>
      <version>1.3.2</version>
    </dependency>
    <dependency>
      <groupId>com.amplifyframework</groupId>
      <artifactId>core</artifactId>
      <version>local</version>
    </dependency>
  </dependencies>
</project>
```

Reply:
I'll check again after publishing to Sonatype as well, to confirm.

Reply:
Thanks for the speedy and effective resolution @richardmcclellan -- and sorry for my bug!